### DISCUSSION TITLE
eForms SDK 1.9.0 released

### DISCUSSION THREAD

**Discussion by bertrand-lorentz** [OP-TED/eForms-SDK#713] [Announcements]
We have released version 1.9.0 of the [eForms Software Development Kit](https://github.com/OP-TED/eForms-SDK/releases/tag/1.9.0).

This release brings various changes and corrections in the metadata content.

Please see the [release notes](https://github.com/OP-TED/eForms-SDK/releases/tag/1.9.0) for more details.

The release artifact is available for [download in Maven Central](https://central.sonatype.com/artifact/eu.europa.ted.eforms/eforms-sdk/1.9.0).

This SDK version will be activated in the Preview environment on 10/10/2023 and in the Production environment on 11/10/2023.

**Comment by AnnaKSZ** [comment]
I got a e-mail message about the service disruption of product system but I've not got any e-mail message about preview system. So thanks your message!

**Reply by pcolast** [reply]
Dear Anna,

All active Preview users have been notified. 
After sending the email we got back Delayed Delivery notification (Delivery will continue to be attempted) where your email address appears together with 6 other email addresses from your organization. 

Thank you.
Have a nice day,
Stefan

**Reply by AnnaKSZ** [reply]
Dear Stefan! 
Thanks for your reply and info, I'll manage the e-mail problem.
Thanks 
Anna
